variables.yaml

```diff
@@ -101,29 +101,29 @@ netcdf_variables:
     units:        dbar
     coordinates:  time depth latitude longitude
     valid_min:    "0"
-    valid_max:    "2000"
+    valid_max:    "1000"
     positive:      "down"
     reference_datum:  "sea-surface"
     instrument:     "instrument_ctd"
     observation_type: "measured"
-    accuracy:         "1"
-    precision:        "2"
-    resolution:       "0.02"
+    accuracy:         "0.05% FS"
+    precision:        "2" #?
+    resolution:       "0.001% FS"
```

Comment:
What is FS here? Again, feel free drop the precision value if it makes sense for this sensor